### DISCUSSION TITLE
How to forward mouse and keyboard event to underneath windows ?

### DISCUSSION THREAD

**Discussion by Donald2010** [tauri-apps/tauri#11507] [Q&A]
My Tauri application has a transparent child webview window, the window should capture mouse and keyboard event, and at the same time can forwards event to other applications under it, so underneath window can interact with mouse an keybaords as normal.

Fow now, my application can ignore mouse event and let it pass through to other windows with `setIgnoreCursorEvent` api, but it can't respond to the event anymore.

I tried `pointer-events:none` css trick hoping it can forward and pop up at the same time, but it seems wont forward events to other applications.

Is there any way to implement this ?

**Comment by FabianLars** [comment]
> Fow now, my application can ignore mouse event and let it pass through to other windows with setIgnoreCursorEvent api, but it can't respond to the event anymore.

This is not supported yet, see https://github.com/tauri-apps/tauri/issues/6164 for the relevant feature request.

No equivalent for keyboard events was requested yet so feel free to open a feature request in this repo.
